osd-recurring-online-meeting.md

```diff
@@ -1,30 +1,26 @@
 ---
 layout: event
-title:  "Open Source Design Monthly Online Meeting"
+title:  "Open Source Design monthly call"
 date:   2018-07-04
 categories: online meet-up
 eventDate: Every First Wednesday
 location: Online at [https://meet.jit.si/opensourcedesign](https://meet.jit.si/opensourcedesign)
-time:  7:00 PM to ... (Berlin Time)
+time:  19:00-20:00 (Berlin Time)
 status: upcoming
 permalink: /open-source-design-monthly-online-meeting/
 ---
 
 Join us for our monthly get together where we talk about all things Open Source Design. Everyone is welcome.
 
-To see what we talked about in previous meetings, look at:
+To see what we talked about in previous meetings, check out the ["monthly call" category on the forum](https://discourse.opensourcedesign.net/c/meta/monthly-call).
```

Comment:
The only thing is that the category is called "Monthly call", with a capitalized M :) but besides this, looking good. Thanks Jan.

Reply:
``To get notifications of this meeting and other events, you can subscribe to our Open Source Design events calendar: [Open Source Design calendar](https://cloud.nextcloud.com/index.php/apps/calendar/p/MIFAFLFJADIVX63I/Open-Source-Design)``

Reply:
Thoughts?

Reply:
Added :)

Reply:
Looks good to me. @evalica ?